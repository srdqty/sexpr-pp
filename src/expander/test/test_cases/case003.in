(let-syntax
  (
    (thunk  (lambda (stx)
              (mk-stx (list (syntax lambda) (syntax (a))
                        (car (cdr (stx-e stx))))
                      stx)))
  )
  (((lambda (a) (thunk (+ a 1))) 5) 0))
