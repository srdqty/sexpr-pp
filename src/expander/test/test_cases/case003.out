>> (begin (((lambda (a0) (lambda (a1) (+ a0 1))) 5) 0))
>> 
